canonical-websites/www.ubuntu.com
# update news section on /cloud

## Done

* removed old news/events section
* added homepage like news strip with news and possible spotlight (not event like in copy doc)
* if we want events, we would need to add that to insights, I think a spotlight like thing is good enough
* removed cloud section stylesheet

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/)
- compare to the [copy doc](https://docs.google.com/document/d/1mQjVXDcoP8ChOzRdmuI55BdXhbURD53qBNc2bgJhfhg/edit)

## Screenshots

![image](https://user-images.githubusercontent.com/441217/27157473-87d4a6d2-5159-11e7-824b-d6a052e433f5.png)
